drone_localization/configs/settings.py

```
@@ -0,0 +1,31 @@
+    image_size_h: int = 256
+    image_size_w: int = 256
+
+    # Qdrant
+    qdrant_host: str = "localhost"
+    qdrant_port: int = 6333
+    qdrant_collection: str = "satellite_gallery"
+
+    # MinIO
+    minio_endpoint: str = "localhost:9000"
+    minio_access_key: str = "minioadmin"
+    minio_secret_key: str = "minioadmin"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "minio_secret_key"

(S105)